src/components/ActivityLog/ActivityMediaForm.tsx

```diff
@@ -154,7 +174,27 @@ const ActivityPhotoPreview = ({
     >
       <Grid container spacing={2} textAlign='left'>
         <Grid item sm='auto' xs={12}>
-          <PhotoPreview imgUrl={url} includeTrashIcon={false} onTrashClick={onDelete} />
+          <Box position='relative'>
+            <PhotoPreview imgUrl={url} includeTrashIcon={false} onTrashClick={onDelete} />
+            {isHeicPlaceholder && (
+              <Box
+                sx={{
+                  backgroundColor: 'rgba(0, 0, 0, 0.7)',
+                  borderRadius: 1,
+                  bottom: 4,
+                  color: 'white',
+                  fontSize: '10px',
+                  left: 4,
+                  padding: '2px 6px',
+                  position: 'absolute',
+                  right: 4,
+                  textAlign: 'center',
+                }}
+              >
+                {strings.HEIC_PREVIEW}
+              </Box>
```

Comment:
Should we be displaying that this was an HEIC preview? Will the user care what kind of photo was uploaded? I'm not familiar with the designs, so this might be a question for product/design

Reply:
I think we should display some kind of indicator. I've asked Léann to review the visual changes in this PR.